gap/Parser.gi

```diff
@@ -211,7 +212,16 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
                         current_line := ReadLineWithLineCount( filestream );
                         NormalizeWhitespace( current_line );
                     od;
-                    Append( filter_string, StripBeginEnd( current_line{ [ 1 .. Minimum( [ PositionSublist( current_line, "," ), PositionSublist( current_line, ");" ) ] ) - 1 ] }, " " ) );
+                    current_line_positition_for_filter := Minimum( [ PositionSublist( current_line, "," ), PositionSublist( current_line, ");" ) ] ) - 1;
```

Comment:
Could you please add some comments here, so that on has at least an idea what this is all about? If I didn't have the tag line, I would never have guessed its purpose.